web/src/app/modules/actions/modules/browser-version-checker/components/browser-version-checker.component.ts

```diff
@@ -0,0 +1,61 @@
+import { Component } from '@angular/core';
+import { TranslateService } from '@ngx-translate/core';
+
+@Component({
+  moduleId: module.id.toString(),
+  selector: 'browser-version-checker',
+  templateUrl: 'browser-version-checker.component.html',
+  styleUrls: ['browser-version-checker.component.css']
+})
+
+
+
+export class BrowserVersionChecker {
+
+  private unsupportedBrowser = [
+    { browser: 'IE', minVersion: 11 },
+    { browser: 'MSIE', minVersion: 10 },
+    { browser: 'Chrome', minVersion: 42 },
+    { browser: 'Firefox', minVersion: 44 },
+    { browser: 'Edge', minVersion: 14 },
+    { browser: 'Safari', minVersion: 9 },
+    { browser: 'Opera', minVersion: 29 }];
+
+  constructor(private translate: TranslateService) {
+
+  }
+
+  public get browserOutdated(): boolean {
+    let currentBrowser = this.getBrowser().split('!!');
+    for (let i = 0; i < this.unsupportedBrowser.length; i++) {
+      const element = this.unsupportedBrowser[i];
+      if (element.browser === currentBrowser[0]) {
+        if (element.minVersion >= parseInt(currentBrowser[1])) {
+          return true;
+        }
+      }
+    }
+    return false;
+  }
+
+  // Based on https://stackoverflow.com/a/5918791
```

Comment:
There's lots of magic code here. Isn't there a npm module for this? I'd not, we need at least more verbose variable names.